ecosystem/sep-0010.md

```diff
@@ -273,6 +286,14 @@ A **Server** may choose to issue JWTs for less than all thresholds and based on
 
 A **Server** that needs to support validating accounts that do not exist can require a signature of the master key of the account address for accounts that do not exist.
 
+### Verifying the Client Domain
+
+A web service requirng SEP-10 authentication may want to attribute each HTTP request made to it to a specific **Client**. For example, a web service may want to offer reduced fees for the users of a specific **Client**.
```

Comment:
💡 One typo, and I also think adding the word software in this sentence helps clarify that when the document says client it don't mean a single instance of a client, but the software used by all instances of the client. Technically the abstract already communicates this, but when reading this specific sentence that meaning didn't feel evident.
```suggestion
A web service requiring SEP-10 authentication may want to attribute each HTTP request made to it to a specific **Client** software. For example, a web service may want to offer reduced fees for the users of a specific **Client**.
```